model/job-management/src/main/proto/beam_expansion_api.proto

```diff
@@ -46,6 +46,9 @@ message ExpansionRequest {
   // A namespace (prefix) to use for the id of any newly created
   // components.
   string namespace = 3;
+
+  // (Optional) If this coder is set, overrides the output type of ExternalTransform
```

Comment:
Let's let this be a request that the outputs have the given coders. This way the expansion service can do its best to satisfy the request, and if it can't the caller has one more chance to do its best to interpret the results.

Reply:
Done.